src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -242,6 +242,7 @@ public class ConfigConstants {
         "opendistro_security.compliance.history.write.ignore_users";
     public static final String OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_EXTERNAL_CONFIG_ENABLED =
         "opendistro_security.compliance.history.external_config_enabled";
+    public static final String OPENDISTRO_SECURITY_SOURCE_FIELD_CONTEXT = "_opendistro_security_source_field_context";
```

Comment:
Similar to other headers (L#52) can it use  ```OPENDISTRO_SECURITY_CONFIG_PREFIX```?

Reply:
Ack.

Reply:
Done